server/routerlicious/packages/lambdas/src/scribe/checkpointManager.ts

```diff
@@ -46,17 +47,11 @@ export class CheckpointManager implements ICheckpointManager {
         const dbOps = pending.map((message) => ({ ...message,
             mongoTimestamp: new Date(message.operation.timestamp) }));
         if (dbOps.length > 0) {
-            await this.opCollection
-                .insertMany(dbOps, false)
-                // eslint-disable-next-line @typescript-eslint/promise-function-async
-                .catch((error) => {
-                    // Duplicate key errors are ignored since a replay may cause us to insert twice into Mongo.
-                    // All other errors result in a rejected promise.
-                    if (error.code !== 11000) {
-                        // Needs to be a full rejection here
-                        return Promise.reject(error);
-                    }
-                });
+            await runWithRetry(
+                async () => {return this.opCollection.insertMany(dbOps, false);},
```

Comment:
nit: you can remove the `{return...}` such that it's just `async () => this.opCollection.insertMany(dbOps, false)`

same for the shouldRetry function